src/GoToDef/GoToDefMouseHandler.cs

```diff
@@ -30,9 +30,12 @@ internal sealed class GoToDefKeyProcessorProvider : IKeyProcessorProvider
         public KeyProcessor GetAssociatedProcessor(IWpfTextView view)
         {
             IVsExtensionManager manager = _serviceProvider.GetService(typeof(SVsExtensionManager)) as IVsExtensionManager;
+            if (manager == null)
+                return null;
+
             IInstalledExtension extension;
             manager.TryGetInstalledExtension("GoToDef", out extension);
-            if (manager == null || extension != null)
+            if (extension == null)
```

Comment:
that should be "extension != null" like it was before. The point of this check is to detect that this extension runs along side with full official PPT (which contains this extension with ID "GotToDef") and auto-disable itself then.